Sources/StreamChatUI/VoiceRecording/Views/WaveformView.swift

```diff
@@ -0,0 +1,76 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+/// Displays an interactive waveform visualisation of an audio file.
+open class WaveformView: _View, ThemeProvider {
+    public struct Content: Equatable {
+        var isRecording: Bool
+        var isPlaying: Bool
+        var duration: TimeInterval
+        var currentTime: TimeInterval
+        var location: URL?
+        var waveform: [Float]
+
+        static let initial = Content(
```

Comment:
I think you forgot to make the properties public

```suggestion
    public struct Content: Equatable {
        public var isRecording: Bool
        public var isPlaying: Bool
        public var duration: TimeInterval
        public var currentTime: TimeInterval
        public var location: URL?
        public var waveform: [Float]

        public static let initial = Content(
```

Another detail worth mentioning is that usually, we also manually create the `init` for these structures since they won't be public by default. It is not a big deal for now, but if a customer needs to create the this content structure, it won't be possible